rest_configuration.cpp

```diff
@@ -316,6 +316,13 @@ void DeRestPluginPrivate::initNetworkInfo()
 
                 QString mac = i->hardwareAddress().toLower();

```

Comment:
I was afraid this might be the case, hence the comment above, but I thought it might only be homebridge-hue that would be affected. The problem is the Philips Hue App requires/expects the bridgeid to match the lan mac-address for ssl to work. There may be more apps that expect this in the future when the hue bridge goes https-only. I don't really want to add another flag, but how about making it configurable? (see updated commit)